### PR TITLE
feat: Set many land update operator

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

File: contracts/Rentals.sol [hunk 1]
```diff
@@ -315,6 +315,28 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver, Ree
         emit OperatorUpdated(_contractAddress, _tokenId, _operator, sender);
     }
 
+    /// @notice Set the operator of various individual LANDs inside an Estate
```
Comment (nachomazzara) [inline contracts/Rentals.sol:318]:
```suggestion
    /// @notice Set the operator of LANDs inside an Estate
```

File: contracts/Rentals.sol [hunk 2]
```diff
@@ -315,6 +315,28 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver, Ree
...
+    /// @dev Differently from the update operator role of the estate, when the asset is transfered to the rentals contract,
+    /// LAND update operators set previously through the Estate contract will remain.
```
Comment (nachomazzara) [inline contracts/Rentals.sol:320]: Maybe something like ```suggestion
    /// LAND update operators can be set to assign granular permissions. LAND update operators will remain if they are inside an Estate when it is transferred. They are only cleared once the LAND is transferred. 
```

**Review by nachomazzara — CHANGES_REQUESTED** [review]
Just left a few comments!

File: contracts/Rentals.sol [hunk 3]
```diff
@@ -315,6 +315,28 @@ contract Rentals is NonceVerifiable, NativeMetaTransaction, IERC721Receiver, Ree
...
     }
 
+    /// @notice Set the operator of various individual LANDs inside an Estate
+    /// @dev Differently from the update operator role of the estate, when the asset is transfered to the rentals contract,
+    /// LAND update operators set previously through the Estate contract will remain. 
+    /// Using this function will allow a tenant to override them. 
+    function setManyLandUpdateOperator(
+        address _contractAddress,
+        uint256 _tokenId,
+        uint256[] memory _landTokenIds,
```
[the remark this answers is not in the document]
Comment (fzavalia) [reply contracts/Rentals.sol]: Good idea, pushed the solution.

**Comment by fzavalia** [comment]
@nachomazzara Should we emit an event when this new method is called?
I think it is unnecessary but we are doing it for the other setUpdateOperator function.
Maybe we should remove the other event.

**Comment by nachomazzara** [comment]
> @nachomazzara Should we emit an event when this new method is called? I think it is unnecessary but we are doing it for the other setUpdateOperator function. Maybe we should remove the other event.

We can remove them yes. We can set up alarms and track operator changes by looking through the tx

**Comment by fzavalia** [comment]
> > @nachomazzara Should we emit an event when this new method is called? I think it is unnecessary but we are doing it for the other setUpdateOperator function. Maybe we should remove the other event.
> 
> We can remove them yes. We can set up alarms and track operator changes by looking through the tx

Removed the event and added tests to check that the LAND contract is emitting them instead

**Review by nachomazzara — APPROVED** [review]

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**